client/src/catalog-home.html

```diff
@@ -261,8 +261,30 @@
       },
 
       ready: function() {
-        this.$._featuredElements.generateRequest();
-        this.$._featuredCollections.generateRequest();
+        this.$.featuredElements.generateRequest();
+        this.$.featuredCollections.generateRequest();
+      },
+
+      _featuredElementsResponse: function() {
+        this._featuredElements = this._filterFeatured(this.$.featuredElements.lastResponse.results);
+      },
+
+      _featuredCollectionsResponse: function() {
+        this._featuredCollections = this._filterFeatured(this.$.featuredCollections.lastResponse.results);
+      },
+
+      _filterFeatured: function(candidates) {
+        var featured = [];
+        var owners = {};
+        // Attempt to feature different owners.
+        for (var i = 0; i < candidates.length && featured.length < 3; i++) {
+          if (!owners[candidates[i].owner]) {
+            owners[candidates[i].owner] = true;
+            featured.push(candidates.splice(i, 1)[0]);
```

Comment:
seems a bit odd for the filter function to mutate its input, ..and makes the logic complicated. Doesn't this skip over an entry each time you remove a candidate?